applications/blog/models.py

```diff
@@ -1,3 +1,62 @@
+import os
 from django.db import models
+from multiselectfield import MultiSelectField
+from applications.alumniprofile.models import Profile
+
 
 # Create your models here.
+class Constants:
+    TAGS = (
```

Comment:
I think instead of HardCoded Tags, we could create one more table for them so that we could add more tags later on.. What do you say?

Reply:
Yeah, sounds good!